src/particles/diagnostics/DiagnosticOutput.H

```diff
@@ -11,6 +11,8 @@
 #define IMPACTX_DIAGNOSTIC_OUTPUT_H
 
 #include "particles/ImpactXParticleContainer.H"
+#include <AMReX_SmallMatrix.H>   // for covariance matrix
+#include "particles/CovarianceMatrix.H"
```

Comment:
Sufficient to use the lower, because we use the alias from here:
```suggestion
#include "particles/CovarianceMatrix.H"
```